requirements-dev.txt

```diff
@@ -5,6 +5,8 @@ sure>=1.2.24
 coverage
 freezegun
 flask
-boto3>=1.2.3
-botocore>=1.3.26
-six
\ No newline at end of file
+# botocore 1.3.29 breaks s3 in tests (lambda and s3 tests)
+# so we need to pin a boto3 and botocore revision pair that we know works
```

Comment:
I'm confused. This is an issue with botocore 1.3.29 or an issue with moto not working with that version?